xdynamo/client.py

```diff
@@ -687,14 +738,18 @@ def _add_conditions_from_query(
             params: DynParams,
             dyn_key: DynKey = None,
             filter_key: str = 'FilterExpression',
-            consistent_read: bool | DefaultType = Default
+            consistent_read: bool | DefaultType = Default,
+            reverse: bool | DefaultType = Default
     ):
         if consistent_read is Default:
             consistent_read = self.consistent_reads
 
         if consistent_read:
             params['ConsistentRead'] = True
 
+        if reverse:
+            params['ScanIndexForward'] = False
+
```

Comment:
That would probably be better to check for the injected default the same way as for `consistent_read`. What do you think?